### DISCUSSION TITLE
An example for using mace-mp-0 to relax structure

### DISCUSSION THREAD

**Discussion by kaijiz** [ACEsuit/mace#330] [Q&A] [answered]
Hi, I am a new beginner in Deep Learning  and i want to use MACE-MP model to relax some crystal structures as practice. But i didn't find related examples and don't know how to begin. I wonder if you could give some examples code. Thank you so much!

**Comment by ilyes319** [comment, marked as answer]
Hey @kaijiz,

You can find an example here of how to use mace-mp: https://mace-docs.readthedocs.io/en/latest/examples/foundation_models.html
For more info on how to relax structures, see https://wiki.fysik.dtu.dk/ase/ase/optimize.html
